modules/installation-supported-aws-machine-types.adoc

```diff
@@ -22,11 +23,35 @@ endif::[]
 ifeval::["{context}" == "installing-aws-china-region"]
 :aws-china:
 endif::[]
-
+ifeval::["{context}" == "installing-aws-outposts-remote-workers"]
+:aws-outposts:
+endif::[]
 
 [id="installation-supported-aws-machine-types_{context}"]
 = Supported AWS machine types
 
+ifdef::aws-outposts[]
+AWS Outposts rack catalog includes options supporting the latest generation Intel powered EC2 instance types with or without local instance storage.
+You need to validate first which instance types are configured in your AWS Outposts.
+Use the AWS CLI to get the list of supported instance types
+[source,terminal]
+----
+aws Outposts get-outpost-instance-types --outpost-id <outpost_id> <1>
```

Comment:
I guess the find and replace changed this one

Reply:
Fixed, thanks